akka-projection-jdbc/src/main/scala/akka/projection/jdbc/internal/Dialect.scala

```diff
@@ -48,14 +50,14 @@ private[projection] object DialectDefaults {
       "CURRENT_OFFSET" VARCHAR(255) NOT NULL,
       "MANIFEST" VARCHAR(4) NOT NULL,
       "MERGEABLE" BOOLEAN NOT NULL,
-      "LAST_UPDATED" BIGINT NOT NULL
+      "LAST_UPDATED" BIGINT NOT NULL,
+      PRIMARY KEY("PROJECTION_NAME","PROJECTION_KEY")
      );""",
       // create index
-      s"""CREATE INDEX "PROJECTION_NAME_INDEX" on $table ("PROJECTION_NAME");""",
-      // add primary key
-      s"""ALTER TABLE $table
-       ADD CONSTRAINT "PK_PROJECTION_ID" PRIMARY KEY("PROJECTION_NAME","PROJECTION_KEY");
-    """)
+      s"""CREATE INDEX IF NOT EXISTS "PROJECTION_NAME_INDEX" on $table ("PROJECTION_NAME");""")
+
+  def dropTableStatement(table: String): String =
+    s" DROP TABLE IF EXISTS $table;"
```

Comment:
we must be able to drop the offset in tests, see https://github.com/akka/akka-platform-guide/pull/422#issuecomment-745551261